kevinm23nelson/Travel-Tracker
# Iteration 5 accessibility/general refactoring

What does this PR do?
This PR refactors the project to simplify the code structure and improve readability, ensuring all functions are DRY and follow SRP.

Where should the reviewer start?
The reviewer should start by examining the updated scripts.js file, which contains the main logic for handling user interactions, data fetching, and trip calculations.

How should this be manually tested?
Manually test this by running the project locally, logging in with valid credentials, and verifying that all features (e.g., calculating trip costs, reserving trips, displaying past/upcoming/pending trips) work as expected.

Any background context you want to provide?
The project is a travel tracker application that allows users to log in, view past and upcoming trips, calculate costs for new trips, and reserve new trips. The recent revisions focus on simplifying the code for better readability and maintenance.
